pkg/pillar/cmd/zedagent/handleappInstMetadata.go

```diff
@@ -18,18 +18,15 @@ func handleAppInstMetaDataModify(ctxArg interface{}, key string,
 }
 
 func handleAppInstMetaDataDelete(ctxArg interface{}, key string, statusArg interface{}) {
-	appInstMetaData := statusArg.(types.AppInstMetaData)
 	ctx := ctxArg.(*zedagentContext)
-	uuidStr := appInstMetaData.Key()
-	PublishAppInstMetaDataToZedCloud(ctx, uuidStr, nil, appInstMetaData.Type, ctx.iteration)
+	PublishAppInstMetaDataToZedCloud(ctx, nil, true)
```

Comment:
This doesn't work since we need to pass &appInstMetaData and is_delete=true so it can send a protobuf message with the uuid and type, but no other content, to indicate that the object is being deleted.